plugins/env/config/routes.rb

```diff
@@ -6,6 +6,9 @@
     end
   end
 
+  get '/external_environment_variable_groups(:format)',
```

Comment:
this already exists from the `resources` block above

Reply:
isn't that only for `environment_variable_groups` ? .. not the _external_ ones?

Reply:
ah yes ... I pushed a commit to clean up the routes :)